gulpfile.js

```diff
@@ -54,6 +54,6 @@ function hugoProd(callback) {
   callback();
 }
 
-exports.build = series(envProd, sass, js.transpileJS, js.concatJS, hugoProd);
```

Comment:
Cool, make sense. Quick change. If you're removing the call to `hugoProd` here in this function argument, I'd say go ahead and remove the function definition on line 52 above since it's not being used anywhere else. 👍